mods/ctf/ctf_leave_message/init.lua

```diff
@@ -0,0 +1,20 @@
+function ctf.send_leave_message(name, timed_out)
+	local player = ctf.player(name)
+
+	local tcolor = ctf_colors.get_color(player).css
+
+	local announcement = "*** " ..  minetest.colorize(tcolor, name) .. " left the game."
+	if timed_out then
+		announcement = announcement .. " (timed out)"
+	end
+	core.chat_send_all(announcement)
+end
+
+function core.send_leave_message()
+	return
+end
```

Comment:
This would make the code shorter. There's no need to register another function; you can simply override the `core.send_leave_message` function.
```suggestion
function core.send_leave_message(player_name, timed_out)
	local player = ctf.player(player_name)
	local tcolor = ctf_colors.get_color(player).css

	local announcement = "*** " ..  minetest.colorize(tcolor, player_name) .. " left the game."
	if timed_out then
		announcement = announcement .. " (timed out)"
	end
	minetest.chat_send_all(announcement)
end
```